docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -828,7 +828,7 @@ paths:
               The flow id of the request, which is written into the logs and passed to called services. Helpful
               for operational troubleshooting and log analysis.
             type: string
-          - name: cursors
+          - name: cursors/
```

Comment:
why "cursors/" ?

Reply:
I don't know how I changed this. Will revert

Reply:
reverted.